Assets/Scripts/Music/MusicController.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MusicController : MonoBehaviour
+{
+    [SerializeField] private AudioClip startMusic;
+    [SerializeField] private AudioClip audio;
+    [SerializeField] private List<AudioClip> audioclips;
+    [SerializeField] private AudioSource audioSource;
+
+    private bool activeMusic;
+
+    private void Start()
+    {
+        audioSource.clip = startMusic;
+        audioSource.Play();
+    }
+    
+    public void ChangeMusicToChase()
+    {
+        audioSource.Stop();
+        audio = audioclips[1];
```

Comment:
```suggestion
        audio = audioclips[1];
```
Je zou hiervan een enum kunnen maken en dan de enums een getal uit de lijst zijn.